src/Console/CommandMakeCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Code16\Sharp\Console;
+
+use Illuminate\Support\Str;
+use Illuminate\Console\GeneratorCommand;
+use Symfony\Component\Console\Input\InputOption;
+
+class CommandMakeCommand extends GeneratorCommand
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'sharp:make:list-command';
```

Comment:
I think this command should be name `sharp:make-entity-command` (instead of `list`) to be consistent

Reply:
Yup, that makes sense. Have made that change.